shell/platform/linux/fl_engine.cc

```diff
@@ -323,6 +359,35 @@ gboolean fl_engine_start(FlEngine* self, GError** error) {
     return FALSE;
   }
 
+  const gchar* const* languages = g_get_language_names();
+  g_autoptr(GPtrArray) locales = g_ptr_array_new_with_free_func(g_free);
+  g_autoptr(GPtrArray) locale_strings = g_ptr_array_new_with_free_func(g_free);
```

Comment:
Maybe comment that this is to own all the strings until after FlutterEngineUpdateLocales is called; I had started a comment below asking if the strings were being leaked until I figured out what was going on.